docs/components/discovery.md

```diff
@@ -33,7 +38,7 @@ class MyClass
     protected $httpClient;
 
     /**
-     * @param HttpClient|null $httpClient Client to do HTTP requests, if not set, autodiscovery will be used to find a HTTP client.
+     * @param HttpClient|null $httpClient
```

Comment:
Not sure if it's an addition. The example code is pretty clean I think. I can add it back if you think it adds any value.